cyberark-secrets-provider-for-kubernetes/ci/test-values-template.yaml

```diff
@@ -0,0 +1,25 @@
+# K8s permission defaults
+rbac:
+  roleName: {{ SECRETS_PROVIDER_ROLE }}
+  roleBindingName: {{ SECRETS_PROVIDER_ROLE_BINDING }}
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: {{ CREATE_SERVICE_ACCOUNT }}
+    # The name of the service account to use
+    name: {{ SERVICE_ACCOUNT }}
+
+secretsProvider:
+  image: cyberark/secrets-provider-for-k8s
+  tag: 1.0.0
+  name: cyberark-secrets-provider
```

Comment:
No need to put these if you don't override the defaults.

Reply:
We might if we want to test different versions